### ISSUE TITLE
Are les integristes really independents ?

### ISSUE THREAD

**Issue by ericlebihan** [lesintegristes/lesintegristes-theme#37]
Just to be clear I would ask you a question about freedom and indepence.
I don't think we should support any organization, politics party, religious movement, brand, etc. except web standards of the W3C of course, no advertising, no bullshit, no censorship (am I going to far ? :-). 
What is your opinion on this topic ?

**Comment by bpierre** [comment]
I agree, that's why we never add those sharing buttons for example (plus I hate them from a UX perspective). The only exception we made so far is Twitter, but there are no rule wthout exception, right? :-)

We could add this statement in a "about" page.
